Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -251,6 +251,20 @@
     - Passenger
 
+- type: listing
+  id: UplinkSyndicateMailcarrier
+  name: uplink-Syndicate-mail-Carrier-name
+  description: uplink-Syndicate-mail-Carrier-desc
+  icon: { sprite: _Goobstation/Clothing/Uniforms/Jumpsuit/mailman.rsi, state: icon }
+  productEntity: ClothingUniformSyndicateMailCarrier
+  cost:
+    Telecrystal: 5
+  categories:
+  - UplinkJob
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)